src/components/SliderInput/index.ts

```diff
@@ -8,7 +8,7 @@ const CLASS_SLIDER_BAR = CLASS_SLIDER + '-bar';
 const CLASS_SLIDER_HANDLE = CLASS_SLIDER + '-handle';
 const CLASS_SLIDER_ACTIVE = CLASS_SLIDER + '-active';
 
-const IS_CHROME = /Chrome\//.test(navigator.userAgent);
+const IS_CHROME = /Chrome\//.test(globalThis.navigator?.userAgent);
```

Comment:
```suggestion
const IS_CHROME = /Chrome\//.test(navigator?.userAgent || "");
```

Reply:
It doesn't work like that in `node`:

```sh
(base) homepc@home-pc:~$ node
Welcome to Node.js v18.12.1.
Type ".help" for more information.
> IS_CHROME_A = /Chrome\//.test(navigator?.userAgent || "");
Uncaught ReferenceError: navigator is not defined
> IS_CHROME_B = /Chrome\//.test(globalThis.navigator?.userAgent);
false
> 
```

And in browsers it works aswell:

![image](https://github.com/playcanvas/pcui/assets/5236548/755e1f3c-5eb0-46ce-bb10-93f308f168e2)

Reply:
OK, yeah, I see. So I guess I was just trying to eliminate `globalThis` because I was concerned that it's a reasonably new addition to JS (ES2020). In the engine for example, we don't use `globalThis` anywhere except in the JSDoc comments. Instead, we tend to do this:

https://github.com/playcanvas/engine/blob/26a73a1fb6db893936c446a43611ff649290a27c/src/core/platform.js#L19-L20

I don't think Babel will transpile `globalThis` for older browsers.

All this said, this is PCUI and not the engine so it's much less likely to run in older browsers. I'm also wondering whether the `IS_CHROME` test is even still needed since it seems to be being used to get around some old browser problem that might not even be an issue today.

I'll approve + merge for now though.